website/root.tsx

```diff
@@ -33,7 +33,7 @@ const mainClassname = css`
   box-sizing: border-box;
   block-size: 100vh;
   padding: 8px;
-  overflow: hidden;
+  overflow: clip;
```

Comment:
Looks like this breaks the navigation style
![image](https://github.com/adazzle/react-data-grid/assets/1452717/043e885c-ab17-4de7-b977-8c771c3e57cf)